### DISCUSSION TITLE
Why there is a rush for PHP8 in Laravel 9?

### DISCUSSION THREAD

**Discussion by ProteanCode** [laravel/framework#42471] [General]
Since Laravel is a general purpose framework "for web artisans", and not some microservice solution (Lumen is pretty much dead, Symfony works here better as you cannot cut off a lot of core providers from Laravel, ie. a View provider which is useless for json api).

Due to that I believe that many other people out here, will never be able to use it as a lot of composer packages do not bother with PHP8 compatibility and are just fine running on PHP 7.4

I may be wrong, but when L10 will be released I think the download rate will be way lower than L8, the L9 is a middleground for broken packages and gives people time to see a problem

**Comment by cyrildewit** [comment]
I think a major reason for this could be because active support (for PHP 7.4) has stopped 5 months ago and the security support ends at 28 Nov 2022.
